iot_config/src/org.rs

```diff
@@ -89,7 +89,7 @@ pub async fn create_org(
     )
     .bind(&owner)
     .bind(&payer)
-    .fetch_one(&mut txn)
+    .fetch_one(&mut *txn)
```

Comment:
this sufficient? assume we don't need to double-deref this one b/c we're creating it within the context of this function instead of passing it in as mutable reference?

Reply:
It's the difference between owning a `Transaction` and having a `&mut Transaction`.

`Deref` to a `PgConnection` is only implemented for `Transaction`.